react-native-gutenberg-bridge/ios/GutenbergBridgeDelegate.swift

```diff
@@ -63,7 +63,7 @@ public protocol GutenbergBridgeDelegate: class {
     ///     - source: the source from where the picker will get the media
     ///     - callback: A callback block to be called with an upload mediaIdentifier and a placeholder image file url, use nil on both parameters to signal that the action was canceled.
```

Comment:
I know this is still draft but let's not forget to update `callback` definition also.